avocado/utils/partition.py

```diff
@@ -297,3 +297,22 @@ def unmount(self, force=True):
                 else:
                     raise PartitionError(self, "Unable to unmount gracefully",
                                          details)
+
+
+def run_dd(opf, count, ipf='/dev/zero', block_size='1M', args=''):
+    """
+    Run dd command for the specified arguments
+
+    :param opf: path to output file
+    :type opf: str
+    :param count: Number of loops i.e count value
+    :type count: int
+    :param ipf: path to input file
+    :type ipf: str
+    :param block_size: block size suffixed with memory unit
+    :type block_size: str
+    :param args: Other arguments to be passed to dd command
+    :type args: str
+    """
+    process.run('dd if=%s of=%s bs=%s count=%d %s'
+                % (ipf, opf, block_size, count, args))
```

Comment:
If we are not going to make the life of test writers easier, the benefit of having such wrapper for `dd` is zero. I don't see a real advantage in writing this:
```
from avocado.utils import run_dd

run_dd('/dev/null', 1024, '/dev/zero', '1')
```

instead of this:

```
from avocado.utils import process

process.run('dd if=/dev/zero of=/dev/null count=1024 bs=1')
```

**Unless** you plan to make this wrapper something more clever. Like, assuming default values when arguments are not provided (i.e. `if=None` means `if=/dev/zero`, `of=None` means `/dev/null`, `bs=None` means `bs=1`, and so on).

Also, I'd use the same arguments names as in the `dd` command itself. I'm used to it, so I don't want to figure out what does `ipf` or `opf` mean.

And for the additional arguments, I'd make it pythonic and use`**kwargs `, populating the command inside the wrapper.